src/cbc_sdk/workload/vulnerability_assessment.py

```diff
@@ -0,0 +1,902 @@
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Model and Query Classes for Vulnerability Assessment API"""
+
+from cbc_sdk.base import (NewBaseModel, BaseQuery, QueryBuilder, QueryBuilderSupportMixin,
+                          IterableQueryMixin, AsyncQueryMixin)
+from cbc_sdk.errors import ApiError
+import logging
+import time
+
+log = logging.getLogger(__name__)
+
+""" Vulnerability models: """
+
+
+class OrganizationalVulnerability(NewBaseModel):
+    """Model for OrganizationalVulnerability"""
+    urlobject = '/vulnerability/assessment/api/v1/orgs/{0}'
+    swagger_meta_file = "workload/models/paginatedDeviceVulnerabilityList.yaml"
+    url_additional = ""
+
+    def __init__(self, cb):
+        """
+        Initialize the OrganizationalVulnerability object.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+        """
+        super(OrganizationalVulnerability, self).__init__(cb)
+        self._full_init = True
+
+    @classmethod
+    def _query_implementation(cls, cb, **kwargs):
+        """
+        Returns the appropriate query object for this object type.
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            **kwargs (dict): Not used, retained for compatibility.
+
+        Returns:
+            VulnerabilityQuery: The query object
+        """
+        return VulnerabilityQuery(cls, cb)
+
+
+class VulnerabilitySummary(NewBaseModel):
+    """Represents a vulnerability summary."""
+    urlobject = "/vulnerability/assessment/api/v1/orgs/{0}"
+    swagger_meta_file = "workload/models/organizationVulnerabilitySummary.yaml"
+    url_additional = "/summary"
+
+    VALID_SEVERITY = ["CRITICAL", "IMPORTANT", "MODERATE", "LOW"]
+
+    def __init__(self, cb):
+        """
+        Initialize VulnerabilitySummary
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+        """
+        super(VulnerabilitySummary, self).__init__(cb)
+        self._full_init = True
+
+    @classmethod
+    def _query_implementation(cls, cb, **kwargs):
+        """
+        Returns the appropriate query object for this object type.
+
+        Reusing VulnerabilityQuery, because it supports the same criteria fields
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with
+            the server.
+            **kwargs (dict): Not used, retained for compatibility.
+
+        Returns:
+            VulnerabilityQuery: The query object
+        """
+        return VulnerabilityQuery(cls, cb)
+
+    @classmethod
+    def get_vulnerability_summary(cls, cb, severity=None, vcenter_id=None):
```

Comment:
This should include organization in the function name. maybe `get_org_vulnerability_summary`